### PR TITLE
Closes #403 Added JavaScript Mastery

### LINKED ISSUE #403: [Add] JavaScript Mastery
### URL

https://www.youtube.com/@javascriptmastery

### Description of the above provided site

JavaScript Mastery is one the best tutorial channel for Web Development including Web3.
He covers everything form basic landing pages to complex projects.

### Category

Youtube

### Sub Category

Web Development

### PR DESCRIPTION
**Pull request by accodes21**
## Fixes Issue
Adds JavaScript Mastery 

## Note to reviewers
Closes #403 to add JavaScript Mastery in youtube category.

### CODE REVIEW

File: database/youtube/web-development.json [hunk 1]
```diff
@@ -109,5 +109,13 @@
       "category": "youtube",
       "subcategory": "web-development",
       "language": "english"
+    },
+    {
+      "name": "JavaScript Mastery",
+      "description": "This channel provides projects based on JavaScript and it's frameworks like ReactJS, NextJS, NodeJS along with API management. It also covers full stack projects with real life application.",
```
Comment (rupali-codes) [inline database/youtube/web-development.json:115]:
```suggestion
      "description": "This channel provides projects based on JavaScript and it's frameworks like ReactJS, NextJS, NodeJS along with API management. It also covers full stack projects with real life applications.",
```

File: database/youtube/web-development.json [hunk 2]
```diff
@@ -109,5 +109,13 @@
       "category": "youtube",
       "subcategory": "web-development",
       "language": "english"
+    },
+    {
+      "name": "JavaScript Mastery",
+      "description": "This channel provides projects based on JavaScript and it's frameworks like ReactJS, NextJS, NodeJS along with API management. It also covers full stack projects with real life applications.",
```
Comment (CBID2) [inline database/youtube/web-development.json:115]:
```suggestion
      "description": "This channel provides projects based on JavaScript and its frameworks like ReactJS, NextJS, NodeJS as well as API management. It also covers full stack projects with real life applications.",
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hi @accodes21! 😁 I made some suggestions to improve the description. Other than that, well done. 🙂

**Comment by accodes21** [comment]
Made the required changes, its an amazing project💯✨

**Review by CBID2 — APPROVED** [review]
Well done @accodes21! 😄 LGTM! 😄 👍🏾

**Review by rupali-codes — APPROVED** [review]
Thank you for the changes

**Pull request merged**